lib/core/package.json

```diff
@@ -118,7 +118,7 @@
     "pnp-webpack-plugin": "1.6.4",
     "postcss": "^7.0.35",
     "postcss-flexbugs-fixes": "^4.2.1",
-    "postcss-loader": "^4.1.0",
+    "postcss-loader": "npm:@phated/postcss-loader@4.2.0-phated.1",
```

Comment:
@shilman Do you have a sense if this will work for end users? It works in my setup with Yarn, and I believe it is supported by npm too.

Reply:
Why not just release `@storybookjs/postcss-loader` and depend on that directly

Reply:
For transparency: We chatted about this and I believe changing the loader out for a differently-named loader might cause issues for people that are using a yarn resolution for `postcss-loader`.

The `npm:` "alias protocol" was introduced in v6.9 (released March 2019) and I believe it's been in Yarn much longer.

If this causes people trouble in the alpha, we can rollback and then swap it out for something like `@storybookjs/postcss-loader`